src/test/java/org/simulator/stochastic/StochasticTestSuiteTest.java

```diff
@@ -266,7 +262,7 @@ public void testModel() throws IOException {
 
           // Runs the simulation again
           try {
-            runSimulation(sim, obs, orderedArgs);
+            sim.start((Double)orderedArgs.get("time"));
```

Comment:
We really need one central place where the `String` literal `TIME = "time"` should be stored.

Reply:
We can make an enum class for constants that are used in various classes. What do you think about it?

Reply:
Yes, `enum`s are good but if it is an enum for just one item it does not make much sense. Also, it is sometimes a bit tricky to use a `String` name from an enum, in particular if it does not match the enum's variable name. For instance, if the enum item is called `TIME` but it should represent the `String` literal `"time"` then you need to add a variable to that enum object and have to retrieve it, for instance like `MyEnum.TIME.getNameValue()` which is much more cumbersome than just using a `public static final String TIME` value. Also, that enum needs a constructor then, etc.